### DISCUSSION TITLE
ValueError: cannot reshape array of size 7950000 into shape (90000)

### DISCUSSION THREAD

**Discussion by PhD-GOAT** [akaraspt/tinysleepnet#9] [Q&A]
to run at step. 2
python prepare_sleepedf.py

signals = psg_f.readSignal(select_ch_idx).reshape(-1, n_epoch_samples)
ValueError: cannot reshape array of size 7950000 into shape (90000)

**Comment by BruceYang521** [comment]
Have you solved this problem？
